### PR TITLE
Update traefik.toml.j2

### PR DESCRIPTION
**Pull request by fritzstauff**
- [x] Installs cleanly on a fresh build of most recent master branch
- [x] Upgrades cleanly from the most recent release
- [ ] Updates documentation relevant to the changes

Enable http and https by default.

### CODE REVIEW

File: roles/traefik/templates/traefik.toml.j2 [hunk 1]
```diff
@@ -1,6 +1,6 @@
-port = ":80"
 graceTimeOut = 10
 logLevel = "ERROR"
+defaultEntryPoints = ["http", "https"]
```
Comment (siddharthist) [inline roles/traefik/templates/traefik.toml.j2]: [This line appears twice](https://github.com/CiscoCloud/mantl/pull/1378/files#diff-f0aec6dec4496d4f0d42e882b8f1cc71R17), is that intended?

File: roles/traefik/templates/traefik.toml.j2 [hunk 2]
```diff
@@ -13,3 +13,14 @@ domain = "{{ traefik_marathon_domain }}"
...
+[entryPoints]
+  [entryPoints.http]
+  address = ":80"
+  [entryPoints.https]
+  address = ":443"
+    [entryPoints.https.tls]
+      [[entryPoints.https.tls.certificates]]
+      CertFile = "/etc/traefik/certs/traefik-admin.cert.pem"
+      KeyFile = "/etc/traefik/certs/traefik-admin.key.pem"
```
Comment (siddharthist) [inline roles/traefik/templates/traefik.toml.j2]: Correct me if I'm wrong, but I don't think generating these certs is currently part of Mantl. After #1463 is merged, you can use 
```
      CertFile = "/etc/pki/mantl/cert"
      KeyFile = "/etc/pki/mantl/key"
```
(I think)

Comment (fritzstauff) [inline roles/traefik/templates/traefik.toml.j2]: I believe the certs are added here https://github.com/CiscoCloud/mantl/blob/1.0.3/roles/traefik/tasks/main.yml#L19-L28

Comment (siddharthist) [inline roles/traefik/templates/traefik.toml.j2]: Oh, nice! Thanks for pointing that out.

Comment (siddharthist) [inline roles/traefik/templates/traefik.toml.j2]: That's been removed from master: https://github.com/CiscoCloud/mantl/commit/c73b294f673314a84d656d07012620937d487f21

**Comment by fritzstauff** [comment]
Thanks for the updates. I went ahead and made the suggested changes. Confirmed removing the duplicate entry has no affect. Did not get a chance to test the new SSL layout.

File: roles/traefik/templates/traefik.toml.j2 (hunk 1 above)
Comment (fritzstauff) [inline roles/traefik/templates/traefik.toml.j2]: Good catch. I have removed it from the latest commit.

**Comment by siddharthist** [comment]
After deploying this branch, the traefik-admin health check fails: 
```
curl: (7) Failed connect to localhost:8081; Connection refused
```

**Comment by siddharthist** [comment]
When I merge the latest master branch, this deploys successfully with green health checks, but when I try to navigate to the IP of my edge node in my browser (with or without TLS), I get a 404.

**Comment by fritzstauff** [comment]
Curious if the edge-node is listening on 8081  localhost only. 

```
 netstat -plan | grep 8081 | grep -i listen
tcp6       0      0 :::8081                 :::*                    LISTEN      15804/traefik

```
Looking at the health check I would guess it's not binding to your floating IP or a problem with a security group?  Can you run a test similar to what the consul health check does from your desktop changing localhost to the appropriate IP? 

```
curl --insecure --silent --show-error --fail --dump-header /dev/stderr --retry 2 --connect-timeout 5 --max-time 5 https://localhost:8081/health" 

```

If you don't have time I will try and run a build later today.

**Comment by fritzstauff** [comment]
Disregard my last message. I see you are getting a 404. Obviously the port is up and receiving a connection. I will do some more test.

**Comment by siddharthist** [comment]
@fritzstauff Have you had some time to follow up on this?

**Comment by fritzstauff** [comment]
@siddharthist Have not figured out why the new certs fail. Got time for more testing today.

**Comment by fritzstauff** [comment]
@siddharthist A restart of Traefik solved this for me. Perhaps Traefik is starting before the certs are loaded or not restarting properly.

**Comment by siddharthist** [comment]
@fritzstauff Looks like there is a handler there: https://github.com/CiscoCloud/mantl/blob/f67fefafe94cd0bba7b763cb8771e17171eceb25/roles/traefik/tasks/main.yml#L35-L47

**Comment by fritzstauff** [comment]
@siddharthist I believe I have found this issue. When viewing traefik via mantl ui proxy. The following alert is triggered https://github.com/containous/traefik/blob/master/webui/src/index.html#L25. Chrome then caches the page without proper .css . This is why restarting traefik seemed to clear the issue. No fix yet just an update.  You can verify by viewing source at https://controller.service.consul/traefik/dashboard/#/.

**Comment by fritzstauff** [comment]
Tested reverting Traefik to  v1.0.alpha.9830086790caf40ce30eb9ed5d317917f8157708.  Seems to handle the proxy and ssl fine.

**Comment by fritzstauff** [comment]
Seems to be an issue with the .css files. When proxying through the mant-ui it fails to find the .css in the screen shot below

screen shot 2016-06-20 at 5 38 41 pm

The referenced .css files referenced don't seem to exit

screen shot 2016-06-20 at 5 38 55 pm


When accessed directly there are no problems . Reverting Traefik back to binary v1.0.alpha.9830086790caf40ce30eb9ed5d317917f8157708 resolves this problem. I think we have a bug on our hands.

**Comment by siddharthist** [comment]
@fritzstauff Awesome, let me know when find a fix and I'll happily test again!

Also, I think the UI/CSS issue was unrelated: #1563

**Comment by siddharthist** [comment]
@fritzstauff Any updates?

**Comment by siddharthist** [comment]
Closing in favor of #1714, which points to the new certs.

**Pull request closed without merging**